app/controllers/api/v2/registration_controller.rb

```diff
@@ -62,11 +63,14 @@ def global
           param_group facet_config.host_configuration.api_param_group, facet_config.host_configuration.api_controller
         end
       end
+      param :remote_execution_interface, String, desc: N_("Identifier of the Host interface for Remote execution")
       def host
         begin
           ActiveRecord::Base.transaction do
             find_host
             prepare_host
+            remote_execution_interface
```

Comment:
this should be called only if rex plugin is enabled, otherwise it fails (like in tests)